{"timestamp":1724679803343,"message":"Database error","log.level":"ERROR","logger.name":"api.blueprints.drbCollection","thread.id":139916069383040,"thread.name":"MainThread","process.id":2350,"process.name":"MainProcess","file.name":"/home/runner/work/drb-etl-pipeline/drb-etl-pipeline/api/blueprints/drbCollection.py","line.number":290,"entity.type":"SERVICE"}
